### DISCUSSION TITLE
Chaning the initial version of npm package

### DISCUSSION THREAD

**Discussion by GoodBoyNeon** [semantic-release/semantic-release#2843] [Q&A] [answered]
My npm package is currently at v1.3.0. Now, I decided to use semantic-release. But now that I ran the ci, semantic-release make a release for v1.0.0. How do I tell semantic-release that it should start the versioning from v1.3.0?

**Comment by spulci** [comment, marked as answer]
Starting with v1.0.0 is semantic-release default behaviour. I would try to suggest you to tag your git repo with tag v1.3.0 before starting semantic release for the first time. I'm not sure about it but please try and let us know.

Please refer to this part of documentation:

[Existing version tags](https://semantic-release.gitbook.io/semantic-release/usage/configuration#existing-version-tags)

Always remember that the GIT tag is the main piece of information taken by semantic release to understand your next release version based on your semantic commit messages.

Otherwise you can start with v1.0.0 and force three commit "--allow-empty" starting with "feat(): release alignment": of course, this is not a best practice at all due to the fact that you're forcing a semantic commit that not reflect a real repository change, as well as for the fact you generate three release that have already been released outside the semantic-release flow.

**Reply by travi** [reply]
> Please refer to this part of documentation:
>
> [Existing version tags](https://semantic-release.gitbook.io/semantic-release/usage/configuration#existing-version-tags)

the most likely issue is that the tag for the previous version either does not exist or does not follow the format expected by semantic-release. thanks for linking to our docs @spulci, as that section should have all the details needed to resolve this problem.
